lib/model/unittest/CHierarchicalResultsTest.cc

```diff
@@ -1992,3 +2016,13 @@ CppUnit::Test* CHierarchicalResultsTest::suite() {
 
     return suiteOfTests;
 }
+
+void CHierarchicalResultsTest::setUp() {
+    // Enable trace level logging for these unit tests
+    ml::core::CLogger::instance().setLoggingLevel(ml::core::CLogger::E_Trace);
```

Comment:
I'm not convinced it's worth committing this and the associated teardown to the repo because:

1. Nobody is likely to read the trace logging of this test when run as part of a full end-to-end run
2. Apart from the nightly debug build our CI runs will compile out trace logging altogether